code/game/gamemodes/cult/cult.dm

```diff
@@ -201,37 +190,14 @@
 		to_chat(cult_mind.current, "\red <FONT size = 3><B>An unfamiliar white light flashes through your mind, cleansing the taint of the dark-one and the memories of your time as his servant with it.</B></FONT>")

```

Comment:
Could change "the dark-one" to use [cult.entity_title1] so this is more compatible with the alternate cults. A Fire god isn't likely to be considered dark.